content/security/docs/incidents.md

```diff
@@ -116,7 +116,6 @@ Periodically attacking your own cluster can help you discover vulnerabilities an
 ## Tools
 + [kube-hunter](https://github.com/aquasecurity/kube-hunter), a penetration testing tool for Kubernetes. 
 + [Gremlin](https://www.gremlin.com/product/#kubernetes), a chaos engineering toolkit that you can use to simulate attacks against your applications and infrastructure. 
-+ [kube-forensics](https://github.com/keikoproj/kube-forensics), a Kubernetes controller that triggers a job that collects the state of a running pod and dumps it in an S3 bucket. 
```

Comment:
Can you see whether there is an equivalent of this using Containerd?

Reply:
I attempted to add some updated info (as well as containerd-related updates in general) in the section on Docker commands in the "Capture volatile artifacts..." section. I link to [this blog](https://kubernetes.io/blog/2022/12/05/forensic-container-checkpointing-alpha/) which has some very good context. A couple sub-points:
- Regarding `docker diff` (and the functionality to do the same in kube-forensics), `nerdctl` doesn't implement `diff`, so it doesn't seem trivially replaceable. It would perhaps require doing e.g. overlayfs-level operations. Also, using a read-only root filesystem as we suggest elsewhere should mitigate problems that this would detect.
- As for snapshotting RAM at the container level, kube-forensics doesn't actually do that, but that blog I link to does link to does discuss checkpointing including RAM in containerd/CRI-O.